app/src/main/java/com/omkarmoghe/pokemap/controllers/service/PokemonNotificationService.java

```diff
@@ -133,41 +146,49 @@ public void onEvent(CatchablePokemonEvent event) {
         Location myLoc = new Location("");
         myLoc.setLatitude(location.latitude);
         myLoc.setLongitude(location.longitude);
-        builder.setContentText(catchablePokemon.size() + getString(R.string.notification_service_pokemon_near));
         builder.setStyle(null);
 
+        List<PokemonIdOuterClass.PokemonId> currentFoundSet = new LinkedList<>();
         if(!catchablePokemon.isEmpty()){
             NotificationCompat.InboxStyle inboxStyle = new NotificationCompat.InboxStyle();
-            inboxStyle.setBigContentTitle(catchablePokemon.size() + getString(R.string.notification_service_pokemon_in_area));
             Set<PokemonIdOuterClass.PokemonId> showablePokemonIDs = preffs.getShowablePokemonIDs();
             
             for(CatchablePokemon cp : catchablePokemon){
                 //Only show the notification if the Pokemon is in the preference list
                 if(showablePokemonIDs.contains(cp.getPokemonId())) {
                     Location pokeLocation = new Location("");
+                    PokemonIdOuterClass.PokemonId pokemonId = cp.getPokemonId();
                     pokeLocation.setLatitude(cp.getLatitude());
                     pokeLocation.setLongitude(cp.getLongitude());
                     long remainingTime = cp.getExpirationTimestampMs() - System.currentTimeMillis();
-                    inboxStyle.addLine(cp.getPokemonId().name() + "(" +
+                    inboxStyle.addLine(pokemonId.name() + "(" +
                             TimeUnit.MILLISECONDS.toMinutes(remainingTime) +
                             " "+getString(R.string.minutes)+"," + Math.ceil(pokeLocation.distanceTo(myLoc)) + " "+getString(
                             R.string.meters)+")");
+                    currentFoundSet.add(pokemonId);
                 }
             }
-
             builder.setStyle(inboxStyle);
+            inboxStyle.setBigContentTitle(currentFoundSet.size() + getString(R.string.notification_service_pokemon_in_area));
         }
+        builder.setContentText(currentFoundSet.size() + getString(R.string.notification_service_pokemon_near));
 
         NotificationManager nm = (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
         nm.notify(notificationId,builder.build());
+        if(vibrate){
+            if(previousFoundPokemon != null && !previousFoundPokemon.equals(currentFoundSet)){
+                vibratorManager.vibrate(VIBRATE_PATTERN,-1);
+            }
+            previousFoundPokemon = currentFoundSet;
+        }
```

Comment:
This vibrates when the scan is done and their is a difference in returned results than before.  But this only does the local scan not the same range as the click scan